src/GraphQLParser/ParserContext.Parse.cs

```diff
@@ -1371,29 +1360,21 @@ private GraphQLType ParseType()
     // http://spec.graphql.org/October2021/#TypeExtension
     private GraphQLTypeExtension ParseTypeExtension()
     {
-        ExpectKeyword("extend");
```

Comment:
Bug here: _extend_ should be a part of _TypeExtensionDefinition_ . Location was incorrectly calculated.